### ISSUE TITLE
Wiki Story Angular Material is outdated.

### ISSUE THREAD

**Issue by StefanNieuwenhuis** [angular/angular-cli#6268]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.

```
@angular/cli: 1.0.3
node: 6.10.1
os: win32 x64
@angular/animations: 4.1.2
@angular/common: 4.1.2
@angular/compiler: 4.1.2
@angular/core: 4.1.2
@angular/forms: 4.1.2
@angular/http: 4.1.2
@angular/material: 2.0.0-beta.3
@angular/platform-browser: 4.1.2
@angular/platform-browser-dynamic: 4.1.2
@angular/router: 4.1.2
@angular/cli: 1.0.3
@angular/compiler-cli: 4.1.2
```

### Repro steps.

I want to include Angular Material in my angular-cli project and followed the steps on [this page](https://github.com/angular/angular-cli/wiki/stories-include-angular-material).
I noticed the information is outdated and after following the guide Material doesn't work at all.
So I've updated the wiki page. 
The repo with the changes is found [here](https://github.com/StefanNieuwenhuis/angular-wiki).

### The log given by the failure.

Not available.

### Desired functionality.

I'd like to see my updates for the Wiki implemented so other users of the awesome angular-cli can integrate Angular Material easily.

### Mention any other details that might be useful.

No other details needed.

**Comment by filipesilva** [comment]
Heya @StefanNieuwenhuis, thanks for taking the time to fix this. We generate the wiki from files in this repo, and the particular one you're looking for is https://github.com/angular/angular-cli/blob/master/docs/documentation/stories/include-angular-material.md.

You can make a PR to fix that, and after it's merged it will be in the next release.

**Comment by jesselpalmer** [comment]
@StefanNieuwenhuis Not sure if you are still working on this, but you could basically just copy the steps on the [Angular Material Getting Started](https://material.angular.io/guide/getting-started) page.

**Comment by dave11mj** [comment]
I believe @StefanNieuwenhuis opened a PR for this https://github.com/angular/angular-cli/pull/6273

**Comment by CaerusKaru** [comment]
Can be closed by #7427

**Comment by filipesilva** [comment]
Closed via https://github.com/angular/angular-cli/pull/7427
